massa-bootstrap/src/server.rs

```diff
@@ -14,365 +42,516 @@ use massa_signature::KeyPair;
-                            debug!("awaiting on bootstrap of peer {}", remote_addr);
-                            match tokio::time::timeout(config.bootstrap_timeout.into(), manage_bootstrap(&config, &mut server, data_execution, version, consensus_command_sender, network_command_sender)).await {
-                                Ok(mgmt) => match mgmt {
-                                    Ok(_) => {
-                                        info!("bootstrapped peer {}", remote_addr)
-                                    },
-                                    Err(BootstrapError::ReceivedError(error)) => debug!("bootstrap serving error received from peer {}: {}", remote_addr, error),
-                                    Err(err) => {
-                                        debug!("bootstrap serving error for peer {}: {}", remote_addr, err);
-                                        // We allow unused result because we don't care if an error is thrown when sending the error message to the server we will close the socket anyway.
-                                        let _ = tokio::time::timeout(config.write_error_timeout.into(), server.send(BootstrapServerMessage::BootstrapError { error: err.to_string() })).await;
-                                    },
-                                }
-                                Err(_timeout) => {
-                                    debug!("bootstrap timeout for peer {}", remote_addr);
-                                    // We allow unused result because we don't care if an error is thrown when sending the error message to the server we will close the socket anyway.
-                                    let _ = tokio::time::timeout(config.write_error_timeout.into(), server.send(BootstrapServerMessage::BootstrapError { error: format!("Bootstrap process timedout ({})", format_duration(config.bootstrap_timeout.to_duration())) })).await;
-                                }
-                            }
-
-                        });
-                        massa_trace!("bootstrap.session.started", {"active_count": bootstrap_sessions.len()});
-                    } else {
-                        let _ = match tokio::time::timeout(self.bootstrap_config.write_error_timeout.into(), server.send(BootstrapServerMessage::BootstrapError {
-                            error: "Bootstrap failed because the bootstrap server currently has no slots available.".to_string()
-                        })).await {
-                            Err(_) => Err(std::io::Error::new(std::io::ErrorKind::TimedOut, "bootstrap error no available slots send timed out").into()),
-                            Ok(Err(e)) => Err(e),
-                            Ok(Ok(_)) => Ok(()),
-                        };
-                        debug!("did not bootstrap {}: no available slots", remote_addr);
-                    }
-                }
+                // load cache if absent
+                // if bootstrap_data.is_none() {
```

Comment:
@AurelienFT  this appears to be commented out in by you here: https://github.com/massalabs/massa/commit/b6b40ebb605734b9fa66e3d9e056dbfab8bf5f6c#diff-2f2a0d279d4c015c632b27bdc7bd2fb62cf753fa8811dead19aadc3133666156R113

can we get rid of this commented out code?

Reply:
Yes

Reply:
Done